esp/scm/ws_ecl_client.ecm

```diff
@@ -16,7 +16,7 @@
 ############################################################################## */
 
 #ifndef WS_ECL_CLIENT_API
-#define WS_ECL_CLIENT_API __declspec(dllimport)
+#define WS_ECL_CLIENT_API DECL_EXPORT
```

Comment:
Here should probably be:

#define WS_ECL_CLIENT_API DECL_IMPORT

Because of line 18 "#ifndef WS_ECL_CLIENT_API" any previous declaration will win.

if you make the change I mentioned above in ws_ecl_client.cpp then the cpp file will already have defined DECL_EXPORT and it will win for the library.  But this "DECL_IMPORT" will win for the client.